Server/server.c

```diff
@@ -82,9 +113,48 @@ int main(int argc, char* argv[]) {
 void send_nok() {
 	// TODO
 }
-void ask_areas() {
-// TODO
+
+void send_ok(pid_t sender, int num) {
+	struct message msg_send;
+
+	//init message
+	msg_send.mtype = sender;
+	msg_send.sender = getpid();
+	msg_send.code = OK;
+
+	//send message
+	if (msgsnd(num, &msg_send, sizeof(struct message)-sizeof(long), 0) == -1) {
+		perror("msgsnd");
+		exit(1);
+	}
+	printf("OK message envoyé à %d\n", sender);
+}
+
+void ask_areas(pid_t sender, int num) {
```

Comment:
I would have rather named the function `list_areas()`, because it sends a list, it does not ask.
The function `ask_areas()` will exist in the client